contrib/bump-bitcoin.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Get the directory where the script is located
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Function to display usage information
+usage() {
+    echo "Usage: $0 <commit-sha>"
+    echo "Example: $0 abcdef1234567890abcdef1234567890abcdef12"
+    exit 1
+}
+
+# Function to print error messages
+error_echo() {
+    echo "Error: $1"
+}
+
+# Check if a commit SHA is provided
+if [ -z "${1:-}" ]; then
+    error_echo "No commit SHA provided."
+    usage
+fi
+
+COMMIT_SHA="$1"
+
+# Validate commit SHA format
+if ! [[ "$COMMIT_SHA" =~ ^[a-fA-F0-9]{40}$ ]]; then
+    error_echo "Invalid commit SHA format."
+    usage
+fi
+
+# Ensure required commands are available
+required_commands=("git" "cmake" "ninja" "clang2py")
+for cmd in "${required_commands[@]}"; do
+    if ! command -v "$cmd" &> /dev/null; then
+        error_echo "$cmd is not installed."
+        exit 1
+    fi
+done
+
+# Pull the subtree with the provided commit SHA
+echo "Pulling subtree with commit SHA: $COMMIT_SHA"
+git subtree pull --prefix=depend/bitcoin https://github.com/bitcoin/bitcoin "$COMMIT_SHA" --squash
+
+echo "Subtree pulled successfully."
+
+pushd depend/bitcoin
+
+mkdir -p build
```

Comment:
This line might be skipped, as the following `cmake -B build ...` creates the `build` directory if necessary.

However, explicitly running `mkdir build` (without `-p`) ensures that no `build` directory previously existed.

Reply:
Thanks, fixed by upgrading CMakeLists and simplifying the configuration! https://github.com/stickies-v/py-bitcoinkernel/pull/20